.github/workflows/manual-publish.yml

```diff
@@ -23,68 +24,41 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
For reviewers: Using macos-latest since mac is the lowest common denominator for platforms that supports all the client SDK target frameworks.